anta/input_models/routing/isis.py

```diff
@@ -26,6 +26,16 @@ class ISISInstance(BaseModel):
     """Configured SR data-plane for the IS-IS instance."""
     segments: list[Segment] | None = None
     """List of IS-IS SR segments associated with the instance. Required field in the `VerifyISISSegmentRoutingAdjacencySegments` test."""
+    graceful_restart: bool = True
```

Comment:
Default is False on EOS.
```suggestion
    graceful_restart: bool = False
```